src/pages/progress_reports/DarCloseout.tsx

```diff
@@ -15,6 +17,27 @@ interface DarCloseoutProps {
 export default function DarCloseout(props: DarCloseoutProps): React.JSX.Element {
     const { readOnly, formState, onFormChange, onValidationChange, validation } = props;
 
+    const [allSigningOfficials, setAllSigningOfficials] = useState<SimplifiedDuosUser[]>([]);
+    const [defaultSigningOfficial, setDefaultSigningOfficial] = useState<SimplifiedDuosUser>();
+
+    const displaySigningOfficial = (so: SimplifiedDuosUser) => {
+        const {displayName, email} = so;
+        const nameString = (email !== undefined && email.length > 0) ? `${displayName} (${email})` : displayName;
+        return {displayText: nameString, ...so};
+    }
+
+    useEffect(() => {
+        const init = async () => {
+            const signingOfficials = await User.getSOsForCurrentUser();
+            setAllSigningOfficials(signingOfficials);
+            const closeoutSigningOfficial = signingOfficials.find(so => so.userId === formState.closeoutSigningOfficial?.userId);
+            if (closeoutSigningOfficial !== undefined) {
+                setDefaultSigningOfficial(closeoutSigningOfficial);
+            }
+        }
+        init();
+    }, []);
```

Comment:
It seems reasonable to add the suggested dependency here (formState.closeoutSigningOfficial?.userId) - From some local testing, it doesn't seem to create an infinite loop.